### PR TITLE
Feature: add syntax for alter table in g4 files and unit tests, for 6480 issue.

### PR DESCRIPTION
**Pull request by wgy8283335**
Changes proposed in this pull request:
-  add syntax for alter table in g4 files and unit tests,
-
-

### CODE REVIEW

File: shardingsphere-sql-parser/shardingsphere-sql-parser-dialect/shardingsphere-sql-parser-oracle/src/main/antlr4/imports/oracle/DDLStatement.g4 [hunk 1]
```diff
@@ -37,7 +37,7 @@ alterIndex
     ;
 
 dropTable
-    : DROP TABLE tableName (CASCADE CONSTRAINTS)? (PURGE)?
+    : DROP TABLE tableName
```
Comment (tristaZero) [inline shardingsphere-sql-parser/shardingsphere-sql-parser-dialect/shardingsphere-sql-parser-oracle/src/main/antlr4/imports/oracle/DDLStatement.g4:40]: I remember we add these key words in the previous PR, right?

**Review by tristaZero — CHANGES_REQUESTED** [review]

File: shardingsphere-sql-parser/shardingsphere-sql-parser-dialect/shardingsphere-sql-parser-oracle/src/main/antlr4/imports/oracle/Keyword.g4 [hunk 2]
```diff
@@ -550,3 +550,7 @@ NUMERIC
 FLOAT
     : F L O A T
     ;
+
+TRIGGERS
+    : T R I G G E R S
+    ;
```
Comment (tristaZero) [inline shardingsphere-sql-parser/shardingsphere-sql-parser-dialect/shardingsphere-sql-parser-oracle/src/main/antlr4/imports/oracle/Keyword.g4:556]: A new line below is needed.

File: shardingsphere-sql-parser/shardingsphere-sql-parser-dialect/shardingsphere-sql-parser-oracle/src/main/antlr4/imports/oracle/OracleKeyword.g4 [hunk 3]
```diff
@@ -818,3 +818,11 @@ IDENTIFIER
...
     : W O R K
     ;
+
+CONTAINER_MAP
+    : C O N T A I N E R UL_ M A P
+    ;
+
+CONTAINERS_DEFAULT
+    : C O N T A I N E R S UL_ D E F A U L T
+    ;
```
Comment (tristaZero) [inline shardingsphere-sql-parser/shardingsphere-sql-parser-dialect/shardingsphere-sql-parser-oracle/src/main/antlr4/imports/oracle/OracleKeyword.g4:828]: A new line below is needed.

**Review by tristaZero — CHANGES_REQUESTED** [review]

File: shardingsphere-sql-parser/shardingsphere-sql-parser-dialect/shardingsphere-sql-parser-oracle/src/main/antlr4/imports/oracle/DDLStatement.g4 (hunk 1 above)
Comment (wgy8283335) [reply shardingsphere-sql-parser/shardingsphere-sql-parser-dialect/shardingsphere-sql-parser-oracle/src/main/antlr4/imports/oracle/DDLStatement.g4]: I've fixed this.

File: shardingsphere-sql-parser/shardingsphere-sql-parser-dialect/shardingsphere-sql-parser-oracle/src/main/antlr4/imports/oracle/Keyword.g4 (hunk 2 above)
Comment (wgy8283335) [reply shardingsphere-sql-parser/shardingsphere-sql-parser-dialect/shardingsphere-sql-parser-oracle/src/main/antlr4/imports/oracle/Keyword.g4]: I've fixed this.

File: shardingsphere-sql-parser/shardingsphere-sql-parser-dialect/shardingsphere-sql-parser-oracle/src/main/antlr4/imports/oracle/OracleKeyword.g4 (hunk 3 above)
Comment (wgy8283335) [reply shardingsphere-sql-parser/shardingsphere-sql-parser-dialect/shardingsphere-sql-parser-oracle/src/main/antlr4/imports/oracle/OracleKeyword.g4]: I've fixed this.

**Review by tristaZero — APPROVED** [review]
Nice job

**Pull request merged**